docs/build-from-scratch.md

```diff
@@ -71,6 +71,23 @@ Ark's `Makefile` has a convenience target, `all-build`, that builds the followin
 * darwin-amd64
 * windows-amd64
 
+#### Running the Ark server locally
+
+Running the Ark server locally really speeds up the iterative development of server because the newer image does not have to be built into a container and then re-deployed on the cluster.
+
+To run the server locally, first create your AWS Credentials file. In case of Minio, the file will look something like -
+```console
+$ cat minio-creds
+[default]
+aws_access_key_id = minio
+aws_secret_access_key = minio123
+```
+
+Next up, start the server:
+```
+AWS_SHARED_CREDENTIALS_FILE=minio-creds ark server -n heptio-ark
```

Comment:
Two things that stand out to me:

1) use the `--plugin-dir` flag to be sure that plugins are found locally. `--plugin-dir=./_output/bin/linux/amd64/` is fine if you're iterating locally.
2) It's probably worth mentioning that you can use different secrets depending on the provider you're running against, e.g. `GOOGLE_APPLICATION_CREDENTIALS=credentials-ark-gke ./_output/bin/linux/amd64/ark server --plugin-dir=./_output/bin/linux/amd64/`